reactive/kubernetes_control_plane.py

```diff
@@ -252,21 +257,32 @@ def check_for_upgrade_needed():
     hookenv.status_set("maintenance", "Checking resources")
     is_leader = is_state("leadership.is_leader")
 
+    # migrate to inclusive flags
+    old, new = "kubernetes-master", "kubernetes-control-plane"  # wokeignore:rule=master
+    for flag in get_flags():
+        if flag.startswith(old):
+            new_flag = flag.replace(old, new, 1)
+            clear_flag(flag)
+            set_flag(new_flag)
+
     # migrate to new flags
-    if is_state("kubernetes-master.restarted-for-cloud"):
-        remove_state("kubernetes-master.restarted-for-cloud")
-        set_state("kubernetes-master.cloud.ready")
-    if is_state("kubernetes-master.cloud-request-sent"):
+    if is_state("kubernetes-control-plane.restarted-for-cloud"):
+        remove_state("kubernetes-control-plane.restarted-for-cloud")
+        set_state("kubernetes-control-plane.cloud.ready")
+    if is_state("kubernetes-control-plane.cloud-request-sent"):
         # minor change, just for consistency
-        remove_state("kubernetes-master.cloud-request-sent")
-        set_state("kubernetes-master.cloud.request-sent")
+        remove_state("kubernetes-control-plane.cloud-request-sent")
+        set_state("kubernetes-control-plane.cloud.request-sent")
+    if is_flag_set("kubernetes-control-plane.snaps.installed"):
+        remove_state("kubernetes-node.snaps.installed")
```

Comment:
This is meant to migrate the `k-c-p.snaps.installed` flag to `k-n.snaps.installed`, right?

```suggestion
        remove_state("kubernetes-control-plane.snaps.installed")
```

Reply:
Also, what's the reason for wanting the `kubernetes-node` prefix instead of `k-c-p` like the other flags?